administrator/language/de-DE/plg_multifactorauth_email.ini

```
@@ -0,0 +1,28 @@
+; Joomla! German Translation
+; (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
+; (C) Translation 2008 - 2022 J!German <https://www.jgerman.de>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_MULTIFACTORAUTH_EMAIL="Multi-Faktor-Authentifizierung - Authentifizierungscode per E-Mail"
+PLG_MULTIFACTORAUTH_EMAIL_CONFIG_FORCE_ENABLE_DESC="Sollte der Authentifizierungscode per E-Mail automatisch als Option für <em>alle</em> Benutzer hinzufügt werden? Nützlich, um Benutzern, die den Zugang zu ihrem Hauptauthentifikator verloren haben und keine Kopie der Backup-Codes aufbewahrt haben, eine Ausweichmöglichkeit zu bieten, allerdings auf Kosten eines gewissen Maßes an Kontrolle und Sicherheit."
+PLG_MULTIFACTORAUTH_EMAIL_CONFIG_FORCE_ENABLE_LABEL="Aktivieren erzwingen"
+PLG_MULTIFACTORAUTH_EMAIL_CONFIG_TIMESTEP_120="Zwei Minuten (empfohlen)"
+PLG_MULTIFACTORAUTH_EMAIL_CONFIG_TIMESTEP_180="Drei Minuten"
+PLG_MULTIFACTORAUTH_EMAIL_CONFIG_TIMESTEP_300="Fünf Minuten"
+PLG_MULTIFACTORAUTH_EMAIL_CONFIG_TIMESTEP_30="Eine halbe Minute"
```

```suggestion
PLG_MULTIFACTORAUTH_EMAIL_CONFIG_TIMESTEP_30="30 Sekunden"
```

Sollte ev. auch im Core geändert werden

---

Bitte einen PR für den Core machen

---

Brian meint, die Einheit sollte einheitlich auf Minute basieren
https://github.com/joomla/joomla-cms/pull/38286#issuecomment-1186260860